features/dd-sdk-android-webview/src/main/kotlin/com/datadog/android/webview/WebViewTracking.kt

```diff
@@ -59,7 +62,8 @@ object WebViewTracking {
     fun enable(
         webView: WebView,
         allowedHosts: List<String>,
-        sdkCore: SdkCore = Datadog.getInstance()
+        sdkCore: SdkCore = Datadog.getInstance(),
+        @FloatRange(from = 0.0, to = 100.0) logsSampleRate: Float = 100f
```

Comment:
Since this is a new parameter, if people are using the SdkCore already, the change as is will be seamless. 
cc @ncreated what's your plan for logs sampling in iOS?

Reply:
With lack of global "sample rate" in Logs, it makes most sense to add `logsSampleRate` to `WVT.enable()` 👍. 

What I can propose for iOS is (vs [what we have today](https://github.com/DataDog/dd-sdk-ios/blob/a8bf98945744a96a750822df35cc6f2d5ea533ca/DatadogWebViewTracking/Sources/WebViewTracking.swift#L37-L41)):
```swift
WebViewTracking.enable(
   webView: myWebView,
   hosts: myHosts,
   logsSampleRate: customLogsSampleRate, // defaults to 100
   in: myCustomCore
)
```
In Swift, what @0xnm proposed feels more natural: `logsSampleRate` before `core`. But Swift supports named parameters, so backward compatibility will be preserved.

Reply:
In Kotlin we have named parameters as well, and probably we shouldn't care much about compatibility in Java. So I would say we need to follow what is proposed for iOS. But not a blocker for me.